src/ParsePromos.php

```diff
@@ -46,6 +46,33 @@ public function parse(array &$promos, array $group_reference, array $config = ar
     }
 
     /**
+     * Parse the promotions array
+     *
+     * @param array $promos
+     * @param array $group_reference
+     * @return array
+     */
+    function groups(array $promos, array $group_reference = array()) {
```

Comment:
Can you declare this as public?

Per: https://scrutinizer-ci.com/g/waynestate/parse-promos/inspections/e82e6ac0-d835-496b-ab87-34cbdfb87be0/issues/files/src/ParsePromos.php?status=new&orderField=path&order=asc&honorSelectedPaths=0